src/Expensify.js

```diff
@@ -92,9 +92,15 @@ function Expensify(props) {
     const [isNavigationReady, setIsNavigationReady] = useState(false);
     const [isOnyxMigrated, setIsOnyxMigrated] = useState(false);
     const [isSplashHidden, setIsSplashHidden] = useState(false);
+    const [isCheckingPublicRoom, setIsCheckingPublicRoom] = useState(true);
+
+    useEffect(() => {
+        if (props.isCheckingPublicRoom !== false) return;
```

Comment:
```suggestion
        if (props.isCheckingPublicRoom) return;
```

Reply:
Your suggestion will involve undefined case

Reply:
Ah that's correct, but I think when we clear the data, it will always be set to the default prop value, which is `true` in this case.

Reply:
Correct, so I accepted your suggestion already